source/features/hide-disabled-milestone-sorter.tsx

```diff
@@ -21,5 +17,10 @@ features.add({
 	include: [
 		pageDetect.isMilestone
 	],
-	init
+	exclude: [
+		() => !select.exists('.js-milestone-issues-container')
```

Comment:
@FloEdelmann curious why you didnt do it like this?

You eliminate a whole function.

Just asking.

Side note: Thanks

Reply:
We need to select the element anyway to check if it exists. If it does, we can directly pass the element instead of its selector to `observeElement`, so it doesn't have to do the `select` again.

Reply:
Right. But my question was https://github.com/yakov116/refined-github/pull/4/commits/b1f16963da253f6d91fc08cb01a5d431c463ee7d

By doing the `observeElement` in the init you eliminate the entire extra function.

Reply:
It's still a function, just an anonymous arrow function instead :D

I don't really care where the `select` and `observeElement` calls are, but why should I change it if the arrow function style has no real advantages?

Reply:
I am talking about the hide function.

But you answer still applies
> why should I change it if the arrow function style has no real advantages?

Reply:
Thanks!!

Are you willing to do #5?

Side Note: You made me very happy by sending the PR's

Reply:
I don't know what #5 is supposed to do. But I see that the limit increased from 1 to 100, which doesn't seem like the right way to do the query.

Reply:
https://github.com/sindresorhus/refined-github/issues/2934#issuecomment-604146813

Reply:
You should be able to cherry pick that. Double check the types

Reply:
You should be able to cherry pick that. Double check the types

Reply:
I feel unconfident to submit a PR for this one, because I can't easily reproduce it, and don't really know how the feature is working.

Reply:
Do you have any org. If yes fork refined-github to your org and you will see "2 open pull requests"